charts/kube-prometheus-stack/templates/prometheus/rules-1.14/windows.pod.rules.yaml

```diff
@@ -0,0 +1,58 @@
+{{- /*
+Generated from 'windows.pod.rules' group from https://raw.githubusercontent.com/prometheus-operator/kube-prometheus/main/manifests/kubernetesControlPlane-prometheusRule.yaml
```

Comment:
wrong info in header. also this file is not recreated if `hack/sync_prometheus_rules.py` is run...

Reply:
I explained the reason why it can't be created via the hack scripts in my PR description. Also happy to change the header if you tell me what you expect it to look like...